### PR TITLE
Convert `translationLeaderboard` to chakra

### PR DESCRIPTION
**Pull request by vdusart**
## Description

Convert `TranslationLeaderboard.tsx` file to Chakra

## Related Issue
Close: #8640
Epic: #8632

### CODE REVIEW

File: src/components/TranslationLeaderboard.tsx [hunk 1]
```diff
@@ -143,46 +101,66 @@ const TranslationLeaderboard: React.FC<IProps> = ({
...
+        </RadioCard>
+      </Flex>
+      <Box bg="background" boxShadow={tableBoxShadow} w="full" mb={8}>
+        <Flex
+          bg="grayBackground"
+          textDecoration="none"
+          justifyContent="space-between"
+          alignItems="center"
+          color="text"
+          mb="px"
```
Comment (pettinarip) [inline src/components/TranslationLeaderboard.tsx:143]:
```suggestion
          mb="1px"
```

File: src/components/TranslationLeaderboard.tsx [hunk 2]
```diff
@@ -214,45 +192,116 @@ const TranslationLeaderboard: React.FC<IProps> = ({
...
             }
             return (
-              <Item key={idx}>
+              <Flex
+                textDecoration="none"
+                justifyContent="space-between"
+                alignItems="center"
+                color="text"
+                boxShadow={tableItemBoxShadow}
+                mb="px"
```
Comment (pettinarip) [inline src/components/TranslationLeaderboard.tsx:201]:
```suggestion
                mb="1px"
```

File: src/components/TranslationLeaderboard.tsx [hunk 3]
```diff
@@ -214,45 +192,116 @@ const TranslationLeaderboard: React.FC<IProps> = ({
...
+          w={{ base: "full", lg: "initial" }}
+          justifyContent="center"
+          ml={{ base: "0", lg: "2" }}
+          mr={{ base: "0", lg: "2" }}
+          _hover={{
+            color: "primary",
+            borderColor: "primary",
+          }}
+          onClick={filterAmount === 10 ? showMore : showLess}
+        >
```
Comment (pettinarip) [inline src/components/TranslationLeaderboard.tsx:286]: This should be a button. I know it was not a button before but I think we should fix this. I'd understand if you don't want to do that on this PR. I could create a new issue to tackle that refactor, up to you. LMK what do you think.

Comment (pettinarip) [reply src/components/TranslationLeaderboard.tsx:286]: And we could share these styles with the ones we are using on the `RadioCard`.

I have in mind this:
```tsx
const Button = (props) => {
  return (
    <ChakraButton
      borderRadius="2rem"
      borderWidth="1px"
      borderStyle="solid"
      borderColor="text"
      ...
      ...props
    />
  )
}

// on the RadioCard component
const RadioCard = () => {
  ...
  
  return (
    <Button
      as="label"
      {...checkbox}
      ...
  )
}
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Great job @vdusart. I've tested it and the component is working fine.

I've left a few small issues and an idea to refactor the buttons we are showing that I think are worth implementing.

**Comment by vdusart** [comment]
Ok @pettinarip,
I just implemented the changes you proposed, tell me what you think about it :smile:

**Review by pettinarip — APPROVED** [review]
Thanks @vdusart lgtm!

**Pull request merged**